doc/stdlib/file.rst

```diff
@@ -412,6 +412,9 @@
 
    Convert a path to an absolute path by adding the current directory if necessary.
 
+   .. note::
+       The function calls ``abspath(p1, p2...)`` and ``abspath(joinpath(p1, p2...))`` are equivalent.
+
```

Comment:
If you change the docstring and run `doc/genstdlib.jl` with the newly compiled julia it should sync the doc here automatically.